Firebase/InstanceID/Private/FIRInstanceID+Private.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#import <FirebaseInstanceID/FIRInstanceID.h>
-#import <FirebaseInstanceID/FIRInstanceIDCheckinPreferences.h>
+#import "FIRInstanceID.h"
```

Comment:
It looks like a revert of [the change](https://github.com/firebase/firebase-ios-sdk/pull/3803/files#diff-6c6af05d6b145c06596c62b94d30aa18) I wonder if anything has changed since #3803 so using different import styles for the same header is not an issue any more?

Reply:
That's a good catch - I'll see if I can add a test for `:generate_multiple_pod_projects` to CI